web.go

```diff
@@ -25,12 +25,13 @@ var metrics = stats.New()
 // WebProvider is a provider.Provider implementation that provides the UI.
 // FIXME to be handled another way.
 type WebProvider struct {
-	Address  string `description:"Web administration port"`
-	CertFile string `description:"SSL certificate"`
-	KeyFile  string `description:"SSL certificate"`
-	ReadOnly bool   `description:"Enable read only API"`
-	server   *Server
-	Auth     *types.Auth
+	Address    string `description:"Web administration port"`
+	CertFile   string `description:"SSL certificate"`
+	KeyFile    string `description:"SSL certificate"`
+	ReadOnly   bool   `description:"Enable read only API"`
+	Statistics bool   `description:"Enable more detailed statistics"`
```

Comment:
If this flag were an int, then you could choose how many errors to display...  Perhaps the flag should be called something else in this case...

Setting it to 0 would disable the feature...